internal/bzlmod/go_deps.bzl

```diff
@@ -592,6 +712,10 @@ _module_tag = tag_class(
         ),
         "build_naming_convention": attr.string(doc = """Removed, do not use""", default = ""),
         "build_file_proto_mode": attr.string(doc = """Removed, do not use""", default = ""),
+        "parent_label": attr.label(
```

Comment:
If we only need this on the synthetic tags generated by `from_file`, we should not make it a public attribute.